### PR TITLE
Add RcppParallel cxxflags to makevars

### PR DESCRIPTION
**Pull request by andrjohns**
This PR updates your Makevars to add RcppParallel's CXXFLAGS, which will be needed for compatibility with Windows ARM64

### CODE REVIEW

File: src/Makevars [hunk 1]
```diff
@@ -1,4 +1,5 @@
 PKG_CXXFLAGS += -DRCPP_USE_UNWIND_PROTECT
+KG_CXXFLAGS += $(shell ${R_HOME}/bin/Rscript -e "RcppParallel::CxxFlags()")
```
Comment (asardaes) [inline src/Makevars:2]: There's a typo, an initial P is missing. Also, where did you find this information? [The documentation](https://rcppcore.github.io/RcppParallel/#r_packages) doesn't mention this.

File: src/Makevars.win [hunk 2]
```diff
@@ -1,4 +1,5 @@
 PKG_CXXFLAGS += -DRCPP_PARALLEL_USE_TBB=1
+PKG_CXXFLAGS += $(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" -e "RcppParallel::CxxFlags()")
```
Comment (asardaes) [inline src/Makevars.win:2]: If I understand correctly, [the output will already include the flag](https://github.com/RcppCore/RcppParallel/blob/5aa08f8b546d2fa99372d02d7b6a50344fb09ff3/R/tbb.R#L53) from line 1, so we could just remove the previous line.

**Review by asardaes — COMMENTED** [review]
Thanks for the heads up :)

File: src/Makevars (hunk 1 above)
Comment (andrjohns) [reply src/Makevars]: > There's a typo, an initial P is missing. 

Oh good catch, sorry about that!

> Also, where did you find this information? [The documentation](https://rcppcore.github.io/RcppParallel/#r_packages) doesn't mention this.

The `CxxFlags` function didn't previously have much use for packages, but the TBB headers needs additional compilation flags on Windows ARM64

**Pull request merged**